src/khoj/routers/api_chat.py

```diff
@@ -370,6 +373,55 @@ async def send_rate_limit_message(message: str):
             f"**Using conversation commands:** {', '.join([cmd.value for cmd in conversation_commands])}"
         )
 
+        if ConversationCommand.Reminder in conversation_commands:
+            crontime, inferred_query = await schedule_query(q, location)
+            trigger = CronTrigger.from_crontab(crontime)
+            common = CommonQueryParamsClass(
+                client=websocket.user.client_app,
+                user_agent=websocket.headers.get("user-agent"),
+                host=websocket.headers.get("host"),
+            )
+            scope = websocket.scope.copy()
+            scope["path"] = "/api/chat"
+            scope["type"] = "http"
```

Comment:
Why is the scope type `http`?

Reply:
That's just the type of connection, not http over https. Anyway that code has been changed now